src/libxively/sft/xi_sft_logic.h

```diff
@@ -19,13 +19,15 @@ typedef struct
     void* send_message_user_data;
     const char** updateable_files;
     uint16_t updateable_files_count;
+    int32_t* updateable_files_download_order;
```

Comment:
I think int16_t would be enough, but this comment smells like nitpicking..., so I leave it up to you.

Reply:
yeah, using 32 just to be safe